test/integration/targets/ec2_vpc_egress_igw/tasks/main.yml

```diff
@@ -0,0 +1,83 @@
+---
+- block:
+
+  # ============================================================
+  - name: create a VPC
+    ec2_vpc_net:
+      name: "{{ resource_prefix }}-vpc"
+      state: present
+      cidr_block: "10.232.232.128/26"
+      region: '{{ ec2_region }}'
```

Comment:
We're using YAML anchors now to greatly simplify the AWS credential passing - see e.g. ec2_group tests.